src/main/java/mops/presentation/ExceptionController.java

```diff
@@ -35,9 +38,18 @@ public ModelAndView resolveException(HttpServletRequest request,
                                          Object handler,
                                          Exception ex) {
         log.error("Caught exception from controller:", ex);
+
+        PresentationError presentationError;
+        if (ex instanceof MaxUploadSizeExceededException) {
+            presentationError = new MessagePresentationError("Die maximale Dateigröße wurde überschritten. "
+                    + "Versuche eine kleinere Datei.");
```

Comment:
```suggestion
                    + "Bitte versuche eine kleinere Datei.");
```

Immer höflich bleiben 😉